src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -402,6 +405,50 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
+        internal async Task HandleSaveSkuRecommendationsRequest(
+            SaveSkuRecommendationsParams parameters,
+            RequestContext<SaveSkuRecommendationsResult> requestContext)
+        {
+            const string SkuRecommendationJsonFileExt = ".json";
+            const string SkuRecommendationHtmlFileExt = ".html";
+            string timeStamp = DateTime.UtcNow.ToString("yyyyMMddHHmmss");
+
+            string[] skuRecommendationFileNames = new string[6];
```

Comment:
I see what you're trying to do here but having it as an array of fixed size 6 can be a little confusing... particularly when it's unclear what 6 refers to, and especially further down here when you index into it directly... like `skuRecommendationFileNames[0]`, `skuRecommendationFileNames[1]`, etc... 

Would it be easier to have this as a List instead and just keep using `List.Add()`?